swagger.yaml

```diff
@@ -53,6 +53,10 @@ securityDefinitions:
     type: apiKey
     in: query
     name: image_token
+  watcherAuth:
+    type: apiKey
+    in: header
+    name: Watcher-Authorization
 
 security:
   - userAuth: [admin, user]
```

Comment:
I also think this might be the slightly more idiomatic way of achieving what we're doing with an additional security definition, but I don't see much real upside to putting this in the roles rather than a separate definition.

If you want to investigate it, then go ahead, but I won't hold the PR up for it.